tests/components/google_assistant/test_trait.py

```diff
@@ -929,6 +911,17 @@
         ATTR_ENTITY_ID: 'lock.front_door'
     }
 
+    # Test without pin
+    trt = trait.LockUnlockTrait(hass,
```

Comment:
I do wonder if this is really the right default behaviour. If a pin isn't set up, wouldn't it be better to by default allow the action rather than deny it?

Reply:
This is per Google policy. If you require a pin but none is set, you should raise the error that will tell the user to configure a pin.

Reply:
Ahh OK. Could probably make this configurable, let users explicitly set the pin to `False` if they don't want to have a pin.

Reply:
Google asked me to add this feature.